.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -32,6 +32,35 @@ body:
       placeholder: Steps to reproduce, e.g. "Go to [Matomo-URL] > Click on Visitors > Click Overview"
     validations:
       required: true
+  - type: input
+    id: customer-website-name
+    attributes:
+      label: Customer and Website Name
+      description: Customer Name and Website Name
+    validations:
+      required: false
+  - type: dropdown
+    id: installation
+    attributes:
+      label: On Premise or Cloud
+      multiple: false
+      options:
+        - On Premise
+        - Cloud
```

Is there any language reason to have the words capitalised?

For the on-prem, it should be plural, see https://andrewschoen.medium.com/software-people-of-the-world-stop-saying-on-premise-you-sound-dumb-56692dae871d.

```suggestion
      label: Customer name and Website name
      description: Customer name and Website name
    validations:
      required: false
  - type: dropdown
    id: installation
    attributes:
      label: On premises or Cloud
      multiple: false
      options:
        - On premises
        - Cloud
```